router/indexerCallbacks.go

```diff
@@ -195,22 +195,18 @@ func (r *Router) getProcessMeta(request RouterRequest) {
 		return
 	}
 
-	if procInfo.Envelope.Anonymous {
-		response.Type = "anonymous"
-	} else {
-		response.Type = "poll"
+	response.Type = api.ProcessType{
+		Anonymous:      procInfo.Envelope.Anonymous,
+		Encrypted:      procInfo.Envelope.EncryptedVotes,
+		CostFromWeight: procInfo.Envelope.CostFromWeight,
+		Serial:         procInfo.Envelope.Serial,
+		UniqueValues:   procInfo.Envelope.UniqueValues,
 	}
-	if procInfo.Envelope.EncryptedVotes {
-		response.Type += " encrypted"
-	} else {
-		response.Type += " open"
-	}
-	if procInfo.Envelope.Serial {
-		response.Type += " serial"
-	} else {
-		response.Type += " single"
-	}
-	response.State = models.ProcessStatus(procInfo.Status).String()
+	response.EntityIndex = procInfo.EntityIndex
```

Comment:
I think instead of adding all these fields as new fields, we should use a struct, such as "ProcessHeader". Don't we have that already somewhere?